tensorboard/plugins/npmi/summary.py

```diff
@@ -111,3 +111,34 @@ def npmi_values(tensor, step=None, description=None):
             step=step,
             metadata=metadata.create_summary_metadata(description),
         )
+
+
+def npmi_embeddings(tensor, step=None, description=None):
+    """Write the embedding representations for the annotations in the dataset.
+
+    Arguments:
+      tensor: A `Tensor` of shape (num_annotations, embedding_dimension) and dtype
```

Comment:
I realize none of the other summary functions here have this, so ok to defer, but I'd recommend adding some enforcement logic inside the functions that checks the tensor shape (i.e. it's rank-2 at least) and dtype, as a guard against incorrect usage.